tests/frontend/test_circuitspecs_chip2.py

```diff
@@ -135,17 +135,17 @@ class DummyCircuit(CircuitSpecs):
     decompositions = {"Interferometer": {}, "MZgate": {}}
 
 
-class TestChip0Compilation:
-    """Tests for compilation using the Chip0 circuit specification"""
+class TestChip2Compilation:
+    """Tests for compilation using the Chip2 circuit specification"""
 
     def test_exact_template(self, tol):
         """Test compilation works for the exact circuit"""
         bb = blackbird.loads(Chip2Specs.circuit)
         bb = bb(
-            squeezing_amplitude_0=0.43,
-            squeezing_amplitude_1=0.65,
-            squeezing_amplitude_2=0.543,
-            squeezing_amplitude_3=0.665,
+            squeezing_amplitude_0=1,
```

Comment:
to save effort later, can we just make the non-zero squeezing value a magic number at the top of the file?

Reply:
Done